"UpdateLiveStats" with new account fails

### Expected Behavior
to display live stats with new account 

new account has never been loaded by pokemongo , so no tutorial has been done 

( worked using level 21 account also no tutorial ) 
2016-08-15 07:33:38,811 [UpdateLiveStats] [INFO] [log_stats] Uptime : 0:00:06 | Earned 0 Stardust | +0 XP | 0 XP/h | Visited 0 stops


new account worked when level 1 was reached

2016-08-15 07:41:47,386 [UpdateLiveStats] [INFO] [log_stats] Uptime : 0:00:39 | Earned 200 Stardust | +1,270 XP | 116,036 XP/h | Visited 0 stops


### Actual Behavior

2016-08-15 07:09:51,384 [       cli] [INFO] Ran for 0:00:05
Traceback (most recent call last):
  File "pokecli.py", line 595, in <module>
    main()
  File "pokecli.py", line 162, in main
    report_summary(bot)
  File "pokecli.py", line 200, in report_summary
    logger.info('Total XP Earned: {}  Average: {:.2f}/h'.format(metrics.xp_earned(), metrics.xp_per_hour()))
  File "/DATA/PokemonGo-Bot/pokemongo_bot/metrics.py", line 28, in xp_earned
    return self.xp['latest'] - self.xp['start']
TypeError: unsupported operand type(s) for -: 'NoneType' and 'NoneType'
2016-08-15 07:09:51,419 [sentry.errors] [ERROR] Sentry responded with an error: 'ascii' codec can't decode byte 0x9c in position 1: ordinal not in range(128) (url: https://app.getsentry.com/api/90254/store/)
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/raven/transport/threaded.py", line 174, in send_sync
    super(ThreadedHTTPTransport, self).send(data, headers)
  File "/usr/local/lib/python2.7/dist-packages/raven/transport/http.py", line 47, in send
    ca_certs=self.ca_certs,
  File "/usr/local/lib/python2.7/dist-packages/raven/utils/http.py", line 66, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/local/lib/python2.7/dist-packages/future/backports/urllib/request.py", line 494, in open
    response = self._open(req, data)
  File "/usr/local/lib/python2.7/dist-packages/future/backports/urllib/request.py", line 512, in _open
    '_open', req)
  File "/usr/local/lib/python2.7/dist-packages/future/backports/urllib/request.py", line 466, in _call_chain
    result = func(*args)
  File "/usr/local/lib/python2.7/dist-packages/raven/utils/http.py", line 46, in https_open
    return self.do_open(ValidHTTPSConnection, req)
  File "/usr/local/lib/python2.7/dist-packages/future/backports/urllib/request.py", line 1284, in do_open
    h.request(req.get_method(), req.selector, req.data, headers)
  File "/usr/lib/python2.7/httplib.py", line 1057, in request
    self._send_request(method, url, body, headers)
  File "/usr/lib/python2.7/httplib.py", line 1097, in _send_request
    self.endheaders(body)
  File "/usr/lib/python2.7/httplib.py", line 1053, in endheaders
    self._send_output(message_body)
  File "/usr/lib/python2.7/httplib.py", line 895, in _send_output
    msg += message_body
UnicodeDecodeError: 'ascii' codec can't decode byte 0x9c in position 1: ordinal not in range(128)
2016-08-15 07:09:51,424 [sentry.errors.uncaught] [ERROR] [u"TypeError: unsupported operand type(s) for -: 'NoneType' and 'NoneType'", u'  File "pokecli.py", line 595, in <module>', u'  File "pokecli.py", line 162, in main', u'  File "pokecli.py", line 200, in report_summary', u'  File "pokemongo_bot/metrics.py", line 28, in xp_earned']
Mon 15 Aug 07:09:51 CEST 2016 Pokebot  Stopped.
Press any button or wait 20 seconds to continue.
### Your config.json 
```
{
    "auth_service": "ptc",
    "username": "",
    "password": "",
    "location": ", ",
    "gmapkey": "---",
    "libencrypt_location": "",
    "tasks": [
      {
        "type": "HandleSoftBan"
      },
      {
        "type": "SleepSchedule",
        "config": {
          "enabled": false,
          "time": "10:24",
          "duration":"4:16",
          "time_random_offset": "00:12",
          "duration_random_offset": "00:34"
        }
      },
      {
        "type": "CollectLevelUpReward"
      },
      {
        "type": "IncubateEggs",
        "config": {
          "longer_eggs_first": true
	}
      },
      {
        "type": "UpdateLiveStats",
        "config": {
          "enabled": true,
          "min_interval": 10,
          "stats": ["uptime", "stardust_earned", "xp_earned", "xp_per_hour", "stops_visited"],
          "terminal_log": true,
          "terminal_title": true
        }
      },
      {
        "type": "TransferPokemon"
      },
      {
        "type": "EvolvePokemon",
        "config": {
            "evolve_all": "none",
            "first_evolve_by": "cp",
            "evolve_above_cp": 500,
            "evolve_above_iv": 0.8,
            "logic": "or",
            "evolve_speed": 20,
            "use_lucky_egg": false
        }
      },
      {
        "type": "RecycleItems",
        "config": {
          "min_empty_space": 30,
          "item_filter": {
            "Pokeball":       { "keep" : 25 },
            "Greatball":       { "keep" : 50 },
            "Ultraball":       { "keep" : 100 },
            "Potion":         { "keep" : 1 },
            "Super Potion":   { "keep" : 2 },
            "Hyper Potion":   { "keep" : 20 },
            "Max Potion":   { "keep" : 30 },
            "Revive":         { "keep" : 20 },
            "Razz Berry":     { "keep" : 20 }
          }
        }
      },
      {
        "type": "CatchVisiblePokemon"
      },
      {
        "type": "CatchLuredPokemon"
      },
      {
              "type": "SpinFort"
      },
      {
        "type": "MoveToMapPokemon",
        "config": {
          "address": "http://:5000",
          "max_distance": 50,
          "min_time": 60,
          "min_ball": 20,
          "prioritize_vips": true,
          "snipe": true,
          "snipe_high_prio_only": true,
          "snipe_high_prio_threshold": 1,
          "update_map": true,
          "mode": "prioritize_vips",
          "catch": {
            "==========Legendaries==========": 0,
            "Aerodactyl": 1000,
            "Snorlax": 1000,
            "Articuno": 1000,
            "Zapdos": 1000,
            "Moltres": 1000,
            "Dratini": 1000,
            "Dragonair": 1000,
            "Dragonite": 1000,
            "Mewtwo": 1000,
            "Mew": 1000,

            "==========Region Locked==========": 0,
            "Farfetch'd": 1000,
            "Kangaskhan": 1000,
            "Mr. Mime": 1000,
            "Tauros": 1000,

            "==========Very Rare==========": 0,
            "Lapras": 900,
            "Electabuzz": 900,
            "Magmar": 900,
            "Ditto": 900,
            "Nidoqueen": 900,


            "==========Starters==========": 0,
            "Bulbasaur": 400,
            "Ivysaur": 600,
            "Venusaur": 1000,

            "Charmander": 400,
            "Charmeleon": 600,
            "Charizard": 1000,

            "Squirtle": 400,
            "Wartortle": 600,
            "Blastoise": 1000,

            "Pikachu": 600,
            "Raichu": 1000,

            "==========Semi Rare==========": 0,
            "Porygon": 200,
            "Scyther": 200,
            "Jynx": 200,

            "==========Uncommon==========": 0,

            "Omanyte": 150,
            "Omastar": 500,

            "Seel": 300,
            "Dewgong": 500,

            "Grimer": 200,
            "Muk": 500,

            "Shellder": 200,
            "Cloyster": 500,

            "Gastly": 200,
            "Haunter": 500,
            "Gengar": 1000,

            "Onix": 600,

            "Drowzee": 600,

            "Hypno": 600,

            "Vulpix": 200,
            "Ninetales": 600,

            "Paras": 100,
            "Parasect": 500,

            "Growlithe": 200,
            "Arcanine": 700,

            "Tentacool": 200,
            "Tentacruel": 500,

            "Mankey": 150,
            "Primeape": 500,

            "Clefairy": 150,
            "Clefable": 500,

            "Jigglypuff": 150,
            "Wigglytuff": 500,

            "Venonat": 100,
            "Venomoth": 500,

            "Diglett": 200,
            "Dugtrio": 500,

            "Meowth": 250,
            "Persian": 500,

            "Psyduck": 150,
            "Golduck": 500,

            "Geodude": 100,
            "Graveler": 500,
            "Golem": 800,

            "Eevee": 200,
            "Vaporeon": 800,
            "Jolteon": 800,
            "Flareon": 800,

            "Kabuto": 150,
            "Kabutops": 500,

            "Magikarp": 150,
            "Gyarados": 800,

            "Pinsir": 150,

            "Ponyta": 200,
            "Rapidash": 500,

            "Slowpoke": 200,
            "Slowbro": 500,

            "Magnemite": 250,
            "Magneton": 500,

            "Krabby": 100,
            "Kingler": 500,

            "Voltorb": 200,
            "Electrode": 500,

            "Exeggcute": 250,
            "Exeggcutor": 500,

            "Cubone": 300,
            "Marowak": 800,

            "Hitmonlee": 400,

            "Hitmonchan": 400,

            "Lickitung": 500,

            "Koffing": 200,
            "Weezing": 500,

            "Rhyhorn": 200,
            "Rhydon": 500,

            "Chansey": 800,

            "Tangela": 300,

            "Horsea": 200,
            "Seadra": 600,

            "Goldeen": 150,
            "Seaking": 500,

            "Staryu": 200,
            "Starmie": 800,


            "==========T1 Evolvers==========": 0,
            "Caterpie": 10,
            "Metapod": 10,
            "Butterfree": 500,

            "Weedle": 10,
            "Kakuna": 10,
            "Beedrill": 500,

            "Pidgey": 10,
            "Pidgeotto": 10,
            "Pidgeot": 300,

            "==========T2 Evolvers==========": 0,
            "Nidoran F": 10,
            "Nidorina": 10,
            "Nidoqueen": 1000,

            "Nidoran M": 10,
            "Nidorino": 10,
            "Nidoking": 1000,

            "Oddish": 100,
            "Gloom": 200,
            "Vileplume": 600,

            "Poliwag": 200,
            "Poliwhirl": 400,
            "Poliwrath": 800,

            "Abra": 300,
            "Kadabra": 600,
            "Alakazam": 800,

            "Machop": 150,
            "Machoke": 400,
            "Machamp": 800,

            "Bellsprout": 100,
            "Weepinbell": 400,
            "Victreebel": 800,

            "==========Trash==========": 0,

            "Rattata": 10,
            "Raticate": 10,

            "Spearow": 10,
            "Fearow": 10,

            "Ekans": 10,
            "Arbok": 10,

            "Sandshrew": 10,
            "Sandslash": 10,

            "Zubat": 10,
            "Golbat": 10,

            "Doduo": 10,
            "Dodrio": 10
          }
        }
      },
      {
        "type": "MoveToFort",
        "config":{
          "lure_attraction": true,
          "lure_max_distance": 1000
        }
      },
      {
        "type": "FollowSpiral",
        "config": {
          "diameter": 6,
          "step_size": 140
        }
      }
    ],
    "map_object_cache_time": 2,
    "forts": {
      "avoid_circles": true,
      "max_circle_size": 200
    },
    "websocket_server": false,
    "walk": 6.66,
    "action_wait_min": 1,
    "action_wait_max": 4,
    "debug": false,
    "test": false,
    "health_record": true,
    "location_cache": true,
    "distance_unit": "km",
    "reconnecting_timeout": 15,
    "evolve_captured": "NONE",
    "catch_randomize_reticle_factor": 0.5,
    "catch_randomize_spin_factor": 1.0,
    "min_ultraball_to_keep": 50,
    "logging_color": true,
    "catch": {
      "any": {"catch_above_cp": 0, "catch_above_iv": 0.5, "logic": "or" },

      "// Pokemons with example": { "always_catch": true },
      "// Gets filtered with release parameters": {},

      "// Legendary pokemons (Goes under S-Tier)": {},
      "Lapras": { "always_catch": true },
      "Moltres": { "always_catch": true },
      "Zapdos": { "always_catch": true },
      "Articuno": { "always_catch": true },

      "// S-Tier pokemons (if pokemon can be evolved into tier, list the representative)": {},
      "Mewtwo": { "always_catch": true },
      "Dragonite": { "always_catch": true },
      "Snorlax": { "always_catch": true },
      "// Mew evolves to Mewtwo": {},
      "Mew": { "always_catch": true },
      "Arcanine": { "always_catch": true },
      "Vaporeon": { "always_catch": true },
      "Gyarados": { "always_catch": true },
      "Exeggutor": { "always_catch": true },
      "Muk": { "always_catch": true },
      "Weezing": { "always_catch": true },
      "Flareon": { "always_catch": true },

      "// Growlithe evolves to Arcanine": {},
      "Growlithe": { "always_catch": true },
      "// Dragonair evolves to Dragonite": {},
      "Dragonair": { "always_catch": true },
      "// Grimer evolves to Muk": {},
      "Grimer": { "always_catch": true },

      "// Magikarp evolves to Gyarados": {},
      "Magikarp": { "always_catch": true },
      "// Exeggcute evolves to Exeggutor": {},
      "Exeggcute": { "always_catch": true },
      "// Eevee evolves to many versions, like Vaporeon, Flareon": {},
      "Eevee": { "always_catch": true },

      "// A-Tier pokemons": {},
      "Slowbro": { "always_catch": true },
      "Victreebel": { "always_catch": true },
      "Machamp": { "always_catch": true },
      "Poliwrath": { "always_catch": true },
      "Clefable": { "always_catch": true },
      "Nidoking": { "always_catch": true },
      "Venusaur": { "always_catch": true },
      "Charizard": { "always_catch": true },
      "Golduck": { "always_catch": true },
      "Nidoqueen": { "always_catch": true },
      "Vileplume": { "always_catch": true },
      "Blastoise": { "always_catch": true },
      "Omastar":  { "always_catch": true },
      "Aerodactyl": { "always_catch": true },
      "Golem": { "always_catch": true },
      "Wigglytuff": { "always_catch": true },
      "Dewgong": { "always_catch": true },
      "Ninetales": { "always_catch": true },
      "Magmar": { "always_catch": true },
      "Kabutops": { "always_catch": true },
      "Electabuzz": { "always_catch": true },
      "Starmie": { "always_catch": true },
      "Jolteon": { "always_catch": true },
      "Rapidash": { "always_catch": true },
      "Pinsir": { "always_catch": true },
      "Scyther": { "always_catch": true },
      "Tentacruel": { "always_catch": true },
      "Gengar": { "always_catch": true },
      "Hypno": { "always_catch": true },
      "Pidgeot": { "always_catch": true },
      "Rhydon": { "always_catch": true },
      "Seaking": { "always_catch": true },
      "Kangaskhan": { "always_catch": true }

},
    "catch_throw_parameters": {
      "excellent_rate": 0.1,
      "great_rate": 0.5,
      "nice_rate": 0.3,
      "normal_rate": 0.1,
      "spin_success_rate" : 0.6

    },
    "release": {
      "any": {"release_below_cp": 50, "release_below_iv": 80, "logic": "or" },

      "// Example of always releasing Rattata:": {},
      "// Rattata": {"always_release": true},
      "// Example of keeping 3 stronger (based on CP) Pidgey:": {},
      "// Pidgey": {"keep_best_cp": 3},
      "// Example of keeping 2 stronger (based on IV) Zubat:": {},
      "// Zubat": {"keep_best_iv": 2},
      "// Also, it is working with any": {},
      "any": {"keep_best_iv": 2},
      "// Example of keeping the 2 strongest (based on CP) and 3 best (based on IV) Zubat:": {},
      "// Zubat": {"keep_best_cp": 2, "keep_best_iv": 3},
      "any": {"keep_best_cp": 1, "keep_best_iv": 2},


      "// Legendary pokemons (Goes under S-Tier)": {},
      "Lapras": { "release_below_cp": 1041, "release_below_iv": 0.8, "logic": "and" },
      "Moltres": { "release_below_cp": 1132, "release_below_iv": 0.8, "logic": "and" },
      "Zapdos": { "release_below_cp": 1087, "release_below_iv": 0.8, "logic": "and" },
      "Articuno": { "release_below_cp": 1039, "release_below_iv": 0.8, "logic": "and" },

      "// S-Tier pokemons (if pokemon can be evolved into tier, list the representative)": {},
      "Mewtwo": { "release_below_cp": 1447, "release_below_iv": 0.8, "logic": "and"},
      "Dragonite": { "release_below_cp": 1221, "release_below_iv": 0.8, "logic": "and" },
      "Snorlax": { "release_below_cp": 1087, "release_below_iv": 0.8, "logic": "and" },
      "// Mew evolves to Mewtwo": {},
      "Mew": { "release_below_cp": 1152, "release_below_iv": 0.8, "logic": "and" },
      "Arcanine": { "release_below_cp": 1041, "release_below_iv": 0.8, "logic": "and" },
      "Vaporeon": { "release_below_cp": 984, "release_below_iv": 0.8, "logic": "and" },
      "Gyarados": { "release_below_cp": 938, "release_below_iv": 0.8, "logic": "and" },
      "Exeggutor": { "release_below_cp": 1032, "release_below_iv": 0.8, "logic": "and" },
      "Muk": { "release_below_cp": 909, "release_below_iv": 0.8, "logic": "and" },
      "Weezing": { "release_below_cp": 784, "release_below_iv": 0.8, "logic": "and" },
      "Flareon": { "release_below_cp": 924, "release_below_iv": 0.8, "logic": "and" },

      "// Growlithe evolves to Arcanine": {},
      "Growlithe": { "release_below_cp": 465, "release_below_iv": 0.8, "logic": "and" },
      "// Dragonair evolves to Dragonite": {},
      "Dragonair": { "release_below_cp": 609, "release_below_iv": 0.8, "logic": "and" },
      "// Grimer evolves to Muk": {},
      "Grimer": { "release_below_cp": 448, "release_below_iv": 0.8, "logic": "and" },
      "// Magikarp evolves to Gyarados": {},
      "Magikarp": { "release_below_cp": 91, "release_below_iv": 0.8, "logic": "and" },
      "// Exeggcute evolves to Exeggutor": {},
      "Exeggcute": { "release_below_cp": 384, "release_below_iv": 0.8, "logic": "and" },
      "// Eevee evolves to many versions, like Vaporeon, Flareon": {},
      "Eevee": { "release_below_cp": 376, "release_below_iv": 0.8, "logic": "and" },

      "// A-Tier pokemons": {},
      "Slowbro": { "release_below_cp": 907, "release_below_iv": 0.8, "logic": "and" },
      "Victreebel": { "release_below_cp": 883, "release_below_iv": 0.8, "logic": "and" },
      "Machamp": { "release_below_cp": 907, "release_below_iv": 0.8, "logic": "and" },
      "Poliwrath": { "release_below_cp": 876, "release_below_iv": 0.8, "logic": "and" },
      "Clefable": { "release_below_cp": 837, "release_below_iv": 0.8, "logic": "and" },
      "Nidoking": { "release_below_cp": 864, "release_below_iv": 0.8, "logic": "and" },
      "Venusaur": { "release_below_cp": 902, "release_below_iv": 0.8, "logic": "and" },
      "Charizard": { "release_below_cp": 909, "release_below_iv": 0.8, "logic": "and" },
      "Golduck": { "release_below_cp": 832, "release_below_iv": 0.8, "logic": "and" },
      "Nidoqueen": { "release_below_cp": 868, "release_below_iv": 0.8, "logic": "and" },
      "Vileplume": { "release_below_cp": 871, "release_below_iv": 0.8, "logic": "and" },
      "Blastoise": { "release_below_cp": 888, "release_below_iv": 0.8, "logic": "and" },
      "Omastar":  { "release_below_cp": 780, "release_below_iv": 0.8, "logic": "and" },
      "Aerodactyl": { "release_below_cp": 756, "release_below_iv": 0.8, "logic": "and" },
      "Golem": { "release_below_cp": 804, "release_below_iv": 0.8, "logic": "and" },
      "Wigglytuff": { "release_below_cp": 760, "release_below_iv": 0.8, "logic": "and" },
      "Dewgong": { "release_below_cp": 748, "release_below_iv": 0.8, "logic": "and" },
      "Ninetales": { "release_below_cp": 763, "release_below_iv": 0.8, "logic": "and" },
      "Magmar": { "release_below_cp": 792, "release_below_iv": 0.8, "logic": "and" },
      "Kabutops": { "release_below_cp": 744, "release_below_iv": 0.8, "logic": "and" },
      "Electabuzz": { "release_below_cp": 739, "release_below_iv": 0.8, "logic": "and" },
      "Starmie": { "release_below_cp": 763, "release_below_iv": 0.8, "logic": "and" },
      "Jolteon": { "release_below_cp": 746, "release_below_iv": 0.8, "logic": "and" },
      "Rapidash": { "release_below_cp": 768, "release_below_iv": 0.8, "logic": "and" },
      "Pinsir": { "release_below_cp": 741, "release_below_iv": 0.8, "logic": "and" },
      "Scyther": { "release_below_cp": 724, "release_below_iv": 0.8, "logic": "and" },
      "Tentacruel": { "release_below_cp": 775, "release_below_iv": 0.8, "logic": "and" },
      "Gengar": { "release_below_cp": 724, "release_below_iv": 0.8, "logic": "and" },
      "Hypno": { "release_below_cp": 763, "release_below_iv": 0.8, "logic": "and" },
      "Pidgeot": { "release_below_cp": 729, "release_below_iv": 0.8, "logic": "and" },
      "Rhydon": { "release_below_cp": 782, "release_below_iv": 0.8, "logic": "and" },
      "Seaking": { "release_below_cp": 712, "release_below_iv": 0.8, "logic": "and" },
      "Kangaskhan": { "release_below_cp": 712, "release_below_iv": 0.8, "logic": "and" },

      "// Koffing evolves to Weezing (A-Tier)": {},
      "Koffing": { "release_below_cp": 403, "release_below_iv": 0.8, "logic": "and" },

      "// Below is B-tier and lower pokemons": {},
      "Caterpie": { "release_below_cp": 156, "release_below_iv": 0.8, "logic": "and" },
      "Weedle": { "release_below_cp": 156, "release_below_iv": 0.8, "logic": "and" },
      "Diglett": { "release_below_cp": 158, "release_below_iv": 0.8, "logic": "and" },
      "Metapod": { "release_below_cp": 168, "release_below_iv": 0.8, "logic": "and" },
      "Kakuna": { "release_below_cp": 170, "release_below_iv": 0.8, "logic": "and" },
      "Rattata": { "release_below_cp": 204, "release_below_iv": 0.8, "logic": "and" },
      "Abra": { "release_below_cp": 208, "release_below_iv": 0.8, "logic": "and" },
      "Zubat": { "release_below_cp": 225, "release_below_iv": 0.8, "logic": "and" },
      "Chansey": { "release_below_cp": 235, "release_below_iv": 0.8, "logic": "and" },
      "Pidgey": { "release_below_cp": 237, "release_below_iv": 0.8, "logic": "and" },
      "Spearow": { "release_below_cp": 240, "release_below_iv": 0.8, "logic": "and" },
      "Meowth": { "release_below_cp": 264, "release_below_iv": 0.8, "logic": "and" },
      "Krabby": { "release_below_cp": 276, "release_below_iv": 0.8, "logic": "and" },
      "Sandshrew": { "release_below_cp": 278, "release_below_iv": 0.8, "logic": "and" },
      "Poliwag": { "release_below_cp": 278, "release_below_iv": 0.8, "logic": "and" },
      "Horsea": { "release_below_cp": 278, "release_below_iv": 0.8, "logic": "and" },
      "Gastly": { "release_below_cp": 280, "release_below_iv": 0.8, "logic": "and" },
      "Ekans": { "release_below_cp": 288, "release_below_iv": 0.8, "logic": "and" },
      "Shellder": { "release_below_cp": 288, "release_below_iv": 0.8, "logic": "and" },
      "Vulpix": { "release_below_cp": 290, "release_below_iv": 0.8, "logic": "and" },
      "Voltorb": { "release_below_cp": 292, "release_below_iv": 0.8, "logic": "and" },
      "Geodude": { "release_below_cp": 297, "release_below_iv": 0.8, "logic": "and" },
      "Doduo": { "release_below_cp": 297, "release_below_iv": 0.8, "logic": "and" },
      "Onix": { "release_below_cp": 300, "release_below_iv": 0.8, "logic": "and" },
      "Mankey": { "release_below_cp": 307, "release_below_iv": 0.8, "logic": "and" },
      "Pikachu": { "release_below_cp": 309, "release_below_iv": 0.8, "logic": "and" },
      "Magnemite": { "release_below_cp": 312, "release_below_iv": 0.8, "logic": "and" },
      "Tentacool": { "release_below_cp": 316, "release_below_iv": 0.8, "logic": "and" },
      "Paras": { "release_below_cp": 319, "release_below_iv": 0.8, "logic": "and" },
      "Jigglypuff": { "release_below_cp": 321, "release_below_iv": 0.8, "logic": "and" },
      "Ditto": { "release_below_cp": 321, "release_below_iv": 0.8, "logic": "and" },
      "Staryu": { "release_below_cp": 326, "release_below_iv": 0.8, "logic": "and" },
      "Charmander": { "release_below_cp": 333, "release_below_iv": 0.8, "logic": "and" },
      "Goldeen": { "release_below_cp": 336, "release_below_iv": 0.8, "logic": "and" },
      "Squirtle": { "release_below_cp": 352, "release_below_iv": 0.8, "logic": "and" },
      "Cubone": { "release_below_cp": 352, "release_below_iv": 0.8, "logic": "and" },
      "Venonat": { "release_below_cp": 360, "release_below_iv": 0.8, "logic": "and" },
      "Bulbasaur": { "release_below_cp": 374, "release_below_iv": 0.8, "logic": "and" },
      "Drowzee": { "release_below_cp": 374, "release_below_iv": 0.8, "logic": "and" },
      "Machop": { "release_below_cp": 381, "release_below_iv": 0.8, "logic": "and" },
      "Psyduck": { "release_below_cp": 386, "release_below_iv": 0.8, "logic": "and" },
      "Seel": { "release_below_cp": 386, "release_below_iv": 0.8, "logic": "and" },
      "Kabuto": { "release_below_cp": 386, "release_below_iv": 0.8, "logic": "and" },
      "Bellsprout": { "release_below_cp": 391, "release_below_iv": 0.8, "logic": "and" },
      "Omanyte": { "release_below_cp": 391, "release_below_iv": 0.8, "logic": "and" },
      "Kadabra": { "release_below_cp": 396, "release_below_iv": 0.8, "logic": "and" },
      "Oddish": { "release_below_cp": 400, "release_below_iv": 0.8, "logic": "and" },
      "Dugtrio": { "release_below_cp": 408, "release_below_iv": 0.8, "logic": "and" },
      "Rhyhorn": { "release_below_cp": 412, "release_below_iv": 0.8, "logic": "and" },
      "Clefairy": { "release_below_cp": 420, "release_below_iv": 0.8, "logic": "and" },
      "Slowpoke": { "release_below_cp": 424, "release_below_iv": 0.8, "logic": "and" },
      "Pidgeotto": { "release_below_cp": 427, "release_below_iv": 0.8, "logic": "and" },
      "Farfetch'd": { "release_below_cp": 441, "release_below_iv": 0.8, "logic": "and" },
      "Poliwhirl": { "release_below_cp": 468, "release_below_iv": 0.8, "logic": "and" },
      "Nidorino": { "release_below_cp": 480, "release_below_iv": 0.8, "logic": "and" },
      "Haunter": { "release_below_cp": 482, "release_below_iv": 0.8, "logic": "and" },
      "Nidorina": { "release_below_cp": 489, "release_below_iv": 0.8, "logic": "and" },
      "Graveler": { "release_below_cp": 501, "release_below_iv": 0.8, "logic": "and" },
      "Beedrill": { "release_below_cp": 504, "release_below_iv": 0.8, "logic": "and" },
      "Raticate": { "release_below_cp": 504, "release_below_iv": 0.8, "logic": "and" },
      "Butterfree": { "release_below_cp": 508, "release_below_iv": 0.8, "logic": "and" },
      "Hitmonlee": { "release_below_cp": 520, "release_below_iv": 0.8, "logic": "and" },
      "Ponyta": { "release_below_cp": 530, "release_below_iv": 0.8, "logic": "and" },
      "Hitmonchan": { "release_below_cp": 530, "release_below_iv": 0.8, "logic": "and" },
      "Charmeleon": { "release_below_cp": 544, "release_below_iv": 0.8, "logic": "and" },
      "Wartortle": { "release_below_cp": 552, "release_below_iv": 0.8, "logic": "and" },
      "Persian": { "release_below_cp": 568, "release_below_iv": 0.8, "logic": "and" },
      "Lickitung": { "release_below_cp": 568, "release_below_iv": 0.8, "logic": "and" },
      "Ivysaur": { "release_below_cp": 571, "release_below_iv": 0.8, "logic": "and" },
      "Electrode": { "release_below_cp": 576, "release_below_iv": 0.8, "logic": "and" },
      "Marowak": { "release_below_cp": 578, "release_below_iv": 0.8, "logic": "and" },
      "Gloom": { "release_below_cp": 590, "release_below_iv": 0.8, "logic": "and" },
      "Porygon": { "release_below_cp": 590, "release_below_iv": 0.8, "logic": "and" },
      "Seadra": { "release_below_cp": 597, "release_below_iv": 0.8, "logic": "and" },
      "Jynx": { "release_below_cp": 600, "release_below_iv": 0.8, "logic": "and" },
      "Weepinbell": { "release_below_cp": 602, "release_below_iv": 0.8, "logic": "and" },
      "Tangela": { "release_below_cp": 607, "release_below_iv": 0.8, "logic": "and" },
      "Fearow": { "release_below_cp": 609, "release_below_iv": 0.8, "logic": "and" },
      "Parasect": { "release_below_cp": 609, "release_below_iv": 0.8, "logic": "and" },
      "Machoke": { "release_below_cp": 614, "release_below_iv": 0.8, "logic": "and" },
      "Arbok": { "release_below_cp": 616, "release_below_iv": 0.8, "logic": "and" },
      "Sandslash": { "release_below_cp": 631, "release_below_iv": 0.8, "logic": "and" },
      "Alakazam": { "release_below_cp": 633, "release_below_iv": 0.8, "logic": "and" },
      "Kingler": { "release_below_cp": 636, "release_below_iv": 0.8, "logic": "and" },
      "Dodrio": { "release_below_cp": 640, "release_below_iv": 0.8, "logic": "and" },
      "Tauros": { "release_below_cp": 643, "release_below_iv": 0.8, "logic": "and" },
      "Primeape": { "release_below_cp": 650, "release_below_iv": 0.8, "logic": "and" },
      "Magneton": { "release_below_cp": 657, "release_below_iv": 0.8, "logic": "and" },
      "Venomoth": { "release_below_cp": 660, "release_below_iv": 0.8, "logic": "and" },
      "Golbat": { "release_below_cp": 672, "release_below_iv": 0.8, "logic": "and" },
      "Raichu": { "release_below_cp": 708, "release_below_iv": 0.8, "logic": "and" },
      "Cloyster": { "release_below_cp": 717, "release_below_iv": 0.8, "logic": "and"},
      "Mr. Mime": { "release_below_cp": 650, "release_below_iv": 0.8, "logic": "and" },


"// Pikachu": {"never_release": true},
"Lapras": {"never_release": true},
        "Moltres": {"never_release": true},
        "Zapdos": {"never_release": true},
        "Articuno": {"never_release": true},
        "Mewtwo": {"never_release": true},
        "Dragonite": {"never_release": true},
        "Snorlax": {"never_release": true},
        "Mew": {"never_release": true},
	"Mewtwo": {"never_release": true},
        "Arcanine": {"never_release": true},
        "Vaporeon": {"never_release": true},
        "Gyarados": {"never_release": true},
        "Exeggutor": {"never_release": true},
        "Muk": {"never_release": true},
        "Weezing": {"never_release": true},
        "Flareon": {"never_release": true},
"Aerodactyl": {"never_release": true},
            "Articuno": {"never_release": true},
            "Dratini": {"never_release": true},
            "Dragonair": {"never_release": true},
            "Dragonite": {"never_release": true},
            "Electabuzz": {"never_release": true},
            "Magmar": {"never_release": true},
            "Ditto": {"never_release": true},
            "Nidoqueen": {"never_release": true},
"Farfetch'd": {"never_release": true},
            "Kangaskhan": {"never_release": true},
            "Mr. Mime": {"never_release": true},
            "Tauros": {"never_release": true},
"Porygon": {"never_release": true},
            "Scyther": {"never_release": true},
            "Jynx": {"never_release": true},


"Rattata": {"always_release": true},
           "// Raticate": {"always_release": true},
            "// Spearow": {"always_release": true},
            "// Fearow": {"always_release": true},
            "Ekans": {"always_release": true},
            "// Arbok": {"always_release": true},
            "Sandshrew": {"always_release": true},
            "// Sandslash": {"always_release": true},
            "Zubat": {"always_release": true},
            "// Golbat": {"always_release": true},
            "Doduo": {"always_release": true},
            "// Dodrio": {"always_release": true}



    },
    "vips" : {
         "Any pokemon put here directly force to use Berry & Best Ball to capture, to secure the capture rate!": {},
        "any": {"catch_above_cp": 1200, "catch_above_iv": 0.9, "logic": "or" },
        "Lapras": {},
        "Moltres": {},
        "Zapdos": {},
        "Articuno": {},

        "// S-Tier pokemons (if pokemon can be evolved into tier, list the representative)": {},
        "Mewtwo": {},
        "Dragonite": {},
        "Snorlax": {},
        "// Mew evolves to Mewtwo": {},
        "Mew": {},
        "Arcanine": {},
        "Vaporeon": {},
        "Gyarados": {},
        "Exeggutor": {},
        "Muk": {},
        "Weezing": {},
        "Flareon": {}

    }
}
```

### Steps to Reproduce

Use fresh PTC account 

mature PTC account works as intended 

### Other Information
OS:4.4.0-28-generic #47-Ubuntu x86_64 x86_64 x86_64 GNU/Linux

Git Commit: 40ea94202d76c46dbc457d83621954d863a36452
Python Version:Python 2.7.12

--- a/CONTRIBUTORS.md
+++ b/CONTRIBUTORS.md
@@ -70,3 +70,4 @@
  * Quantra
  * pmquan
  * net8q
+ * SyncX

--- a/pokemongo_bot/metrics.py
+++ b/pokemongo_bot/metrics.py
@@ -7,15 +7,15 @@ class Metrics(object):
     def __init__(self, bot):
         self.bot = bot
         self.start_time = time.time()
-        self.dust = {'start': None, 'latest': None}
-        self.xp = {'start': None, 'latest': None}
-        self.distance = {'start': None, 'latest': None}
-        self.encounters = {'start': None, 'latest': None}
-        self.throws = {'start': None, 'latest': None}
-        self.captures = {'start': None, 'latest': None}
-        self.visits = {'start': None, 'latest': None}
-        self.unique_mons = {'start': None, 'latest': None}
-        self.evolutions = {'start': None, 'latest': None}
+        self.dust = {'start': 0, 'latest': 0}
+        self.xp = {'start': 0, 'latest': 0}
+        self.distance = {'start': 0, 'latest': 0}
+        self.encounters = {'start': 0, 'latest': 0}
+        self.throws = {'start': 0, 'latest': 0}
+        self.captures = {'start': 0, 'latest': 0}
+        self.visits = {'start': 0, 'latest': 0}
+        self.unique_mons = {'start': 0, 'latest': 0}
+        self.evolutions = {'start': 0, 'latest': 0}
 
         self.releases = 0
         self.highest_cp = {'cp': 0, 'desc': ''}

